### PR TITLE
[Hotfix][Connector-e2e][File-local] Complete the e2e test case env co…

### LINKED ISSUE #4646: [Bug] [Connector-e2e] local-file e2e missing an env configuration
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

图片

### SeaTunnel Version

v2.3.1

### SeaTunnel Config

```conf
null
```

### Running Command

```shell
null
```

### Error Exception

```log
null
```

### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by MonsterChenzhuo**
## Purpose of this pull request
close #4646

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_to_assert.conf [hunk 1]
```diff
@@ -51,22 +66,7 @@ source {
           c_date = date
           c_decimal = "decimal(38, 18)"
           c_timestamp = timestamp
-          c_row = {
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_to_assert.conf:54]: Why delete it?

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_projection_to_assert.conf [hunk 2]
```diff
@@ -42,22 +67,7 @@ source {
           c_date = date
           c_decimal = "decimal(38, 18)"
           c_timestamp = timestamp
-          c_row = {
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_projection_to_assert.conf:45]: Why delete it?

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_projection_to_assert.conf [hunk 3]
```diff
@@ -18,16 +18,41 @@
 ###### This config file is a demonstration of streaming processing in seatunnel config
 ######
 
+env {
```
Comment (TyrantLucifer) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_projection_to_assert.conf:21]: It's duplicate with #4615 https://github.com/apache/incubator-seatunnel/pull/4615/files#diff-ee22b91619cbf4eea14ae402900bf2bb469e4ad4a2a6d1387af1c3db93d5d7fcR21

**Review by TyrantLucifer — COMMENTED** [review]
This pull reqeust dose not make sense, so I will close it, thank you for you contribution.

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_to_assert.conf (hunk 1 above)
Comment (MonsterChenzhuo) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_to_assert.conf:54]: @TyrantLucifer  It's not deletion, but not using the hecon plug-in to format the file content at that time.

**Pull request closed without merging**